rule/enforce-repeated-arg-type-style.go

```diff
@@ -135,13 +126,14 @@ func (r *EnforceRepeatedArgTypeStyleRule) Apply(file *lint.File, arguments lint.
 				var prevType ast.Expr
 				if fn.Type.Params != nil {
 					for _, field := range fn.Type.Params.List {
-						// TODO: For invalid types we could have compared raw import names (import package alias + selector), but will it work properly in all the cases?
-						if !r.isInvalidType(typesInfo.Types[field.Type].Type) && types.Identical(typesInfo.Types[field.Type].Type, typesInfo.Types[prevType].Type) {
+						prevTypeStr := gofmt(prevType)
+						currentTypeStr := gofmt(field.Type)
+						if currentTypeStr == prevTypeStr {
 							failures = append(failures, lint.Failure{
 								Confidence: 1,
-								Node:       field,
+								Node:       prevType,
```

Comment:
It's a deliberated change; the intent is to be coherent with the message ("repeated ... type ... can be omitted") by pointing to the type that it can actually removed/omitted.